### PR TITLE
Added Heroku ReadMe , Reset minor Heroku changes to Default.

### PR DESCRIPTION
**Pull request by chanakaDe**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This includes new documentation on Heroku deployment

* **What is the current behavior?** (You can also link to an open issue here)
Runs as usual

* **What is the new behavior (if this is a feature change)?**
Nothing new other than new documentation and bug fixes 

* **Other information**:
In package.json, the line number of some lines are changed. but not a single code. 👍

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -205,7 +205,7 @@
     "url": "https://github.com/AngularClass/angular-starter/issues"
   },
   "engines": {
-    "node": ">= 4.2.1",
-    "npm": ">= 3"
+    "node": "6.11.3",
```
Comment (liorkesos) [inline package.json]: You are hardcoding the version - this is not good - you should probably use semver based ^ prefix to say higher or equal to 6.11

File: heroku-deploy.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# How to deploy MEAN on Heroku
+
+ 
+First you need to visit https://heroku.com and create a free account. 
+Then you need to install Heroku CLI (https://devcenter.heroku.com/articles/heroku-cli) according to your OS. All the instructions available there on how to install.
+If you have't folk the official MEAN repo yet, folk the repo and then you need to clone it to your local machine.
```
Comment (sabousha) [inline heroku-deploy.md]: "fork" not "folk" !

File: heroku-deploy.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Then navigate to 'MEAN' directory.
+Use following command and then enter your username and password. (Now you are ready to go)
+```
+heroku login
+```
+Now open the MEAN application uing your favourite editor and edit following files.
```
Comment (sabousha) [inline heroku-deploy.md]: "Using" not "uing"

File: heroku-deploy.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  
+After choosing one of the above solutions, you have to update `server-start.js` like this : 
+
+ 
+ ```
+ process.env.MONGO_HOST='mongodb://username:password@ds135574.mlab.com:35574/new-mean'
+ process.env.MONGO_PORT='35574'
+ ```
+ 
+Then make sure you are committing all the changes to your repo. And puch changes.
```
Comment (sabousha) [inline heroku-deploy.md]: "Push" not "puch"

**Pull request closed without merging**